corehq/apps/dump_reload/management/commands/dump_case_data.py

```diff
@@ -0,0 +1,83 @@
+from __future__ import absolute_import
+from __future__ import print_function
+
+from datetime import datetime
+
+from django.db import connections
+from django.conf import settings
+from django.core.management.base import BaseCommand, CommandError
+
+from corehq.apps.dump_reload.const import DATETIME_FORMAT
+from corehq.form_processor.models import CommCareCaseSQL
+
+
+class Command(BaseCommand):
+    help = "Dump all case data, per case type per shard in csv files"
+
+    def add_arguments(self, parser):
+        parser.add_argument('--domain')
+        parser.add_argument('--case_type')
+        parser.add_argument('--db_conn')
+
+    def ensure_params(self, for_domain, for_case_type, for_db_conn):
+        if for_db_conn not in settings.DATABASES:
```

Comment:
could put these in the `choices` param of the argument so that argparser can do the validation and they would also get added to the help text